includes/class-omise-money.php

```diff
@@ -0,0 +1,56 @@
+<?php
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * @since 3.5
+ */
+class Omise_Money {
+	/**
+	 * @var array
+	 */
+	private static $subunit_multiplier = array(
+		'AUD' => 100,
+		'CAD' => 100,
+		'CHF' => 100,
+		'CNY' => 100,
+		'DKK' => 100,
+		'EUR' => 100,
+		'GBP' => 100,
+		'HKD' => 100,
+		'JPY' => 1,
+		'MYR' => 100,
+		'SGD' => 100,
+		'THB' => 100,
+		'USD' => 100
+	);
+
+	/**
+	 * @param  int|float $amount
+	 *
+	 * @return float
+	 */
+	public static function purify_amount( $amount ) {
+		if ( ! is_numeric( $amount ) ) {
+			throw new Exception( __( 'An amount has to be integer, or float.', 'omise' ) );
+		}
+
+		return (float) $amount;
+	}
+
+	/**
+	 * @param  int|float $amount
+	 * @param  string    $currency
+	 *
+	 * @return int
+	 */
+	public function to_subunit( $amount, $currency ) {
+		$amount   = static::purify_amount( $amount );
+		$currency = strtoupper( $currency );
+
+		if ( ! isset( static::$subunit_multiplier[ $currency ] ) ) {
+			throw new Exception( __( 'We do not support the currency you are using.', 'omise' ) );
+		}
+
+		return (int) ( ( floor( $amount * 100 ) / 100 ) * static::$subunit_multiplier[ $currency ] );
+	}
```

Comment:
Still no `subunit_to_amount`? This is necessary in places and this class would seem to be the correct home

Reply:
At the moment, there is no place that needs `subunit_to_amount`. So I didn't have it here, for now.